river/LockSurface.zig

```diff
@@ -54,6 +59,42 @@ pub fn create(wlr_lock_surface: *wlr.SessionLockSurfaceV1) void {
     Subsurface.handleExisting(wlr_lock_surface.surface, .{ .lock_surface = lock_surface });
 }
 
+pub fn destroy(lock_surface: *LockSurface) void {
+    lock_surface.output().lock_surface = null;
+
+    {
+        var surface_it = lock_surface.lock.surfaces.iterator(.forward);
+        const new_focus: Seat.FocusTarget = while (surface_it.next()) |surface| {
+            if (surface == lock_surface.wlr_lock_surface) continue;
+
+            break .{ .lock_surface = fromWlrLockSurface(surface) };
```

Comment:
Because it’s only a single branch of the if? Because it doesn’t split the while with a blank line but also makes it clear what’s inside the condition?

I like this too, just didn’t think of it.